service-worker/windowclient-navigate/index.html

```diff
@@ -0,0 +1,31 @@
+---
+feature_name: "Service Worker: WindowClient.navigate()"
+chrome_version: 49
+feature_id: 5314750808326144
+---
+
+<h3>Background</h3>
+<p>
+  <code>WindowClient.navigate()</code> allows a service worker to cause a web
+  page to navigate to a specific URL.
+</p>
+
+<p>
+  This sample shows how <code>WindowClient.navigate()</code> can be called
+  within a service worker's <code>activate</code> handler, to navigate to
+  <code>activated.html</code>. If your browser supports service workers, and
+  supports <code>WindowClient.navigate()</code>, then you'll be automatically
+  navigated away from this page once the service worker activates.
```

Comment:
Weird. See https://code.google.com/p/chromium/issues/detail?id=500911#c23 and https://www.w3.org/TR/service-workers/#client-navigate-method